requirements/edx/base.txt

```diff
@@ -726,7 +727,7 @@ oauthlib==3.0.1
     #   social-auth-core
 openedx-calc==3.0.1
     # via -r requirements/edx/base.in
-openedx-events==0.9.1
+openedx-events==0.10.0
```

Comment:
One breaking change related to `AvroAttrsBridge` field but that isn't used anywhere within the edx-platform. See [CHANGELOG](https://github.com/openedx/openedx-events/blob/main/CHANGELOG.rst#0100---2022-05-20)

Reply:
I think this was actually used, see https://edx-internal.slack.com/archives/CD9FTF8HW/p1655302059505749